misc/python/materialize/cli/ci_coverage_pr_report.py

```diff
@@ -138,14 +138,16 @@ def get_report(coverage: Coverage) -> str:
                         f.write(line)
                 f.truncate()
 
-        subprocess.run(
-            ["git", "config", "user.email", "coverage@materialize.com"],
-            check=True,
-        )
-        subprocess.run(
-            ["git", "config", "user.name", "Code Coverage"],
-            check=True,
-        )
+        # Only set user for CI, not when testing this script locally
+        if ui.env_is_truthy("CI"):
+            subprocess.run(
+                ["git", "config", "user.email", "coverage@materialize.com"],
```

Comment:
While you are at it, I suggest also adding the `--local` option to `git config` here and below.

```
    --local               use repository config file
```

Reply:
As discussed, that's the default.